.github/workflows/release.yml

```diff
@@ -89,6 +92,14 @@ jobs:
             secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD ;
             secret/data/github/repo/${{ github.repository }}/rancher-prime-stg-registry/credentials username | STAGE_REGISTRY_USERNAME ;
             secret/data/github/repo/${{ github.repository }}/rancher-prime-stg-registry/credentials password | STAGE_REGISTRY_PASSWORD ;
+            secret/data/github/repo/${{ github.repository}}/rancher-prime-registry/credentials username | PRIME_REGISTRY_USERNAME ;
+            secret/data/github/repo/${{ github.repository}}/rancher-prime-registry/credentials password | PRIME_REGISTRY_PASSWORD ;
+            secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials registry | PRIME_REGISTRY;
+
+      - name: Prepare Docker environment to support sbom/provenance parameters
+        run: |
+          sudo bash -c 'echo "{\"features\": {\"containerd-snapshotter\": true}}" >> /etc/docker/daemon.json'
```

Comment:
Curiosity: why do we need to invoke Bash explicitly here?
Wouldn't this do the same, or is it eg. a matter of (another) shell vs Bash?
```
          echo '{"features": {"containerd-snapshotter": true}}' >> /etc/docker/daemon.json
```

Reply:
It's because of using `sudo`.
Using `sudo echo ... > /path/to/file`, writing to that file will not use the elevated privileges provided by sudo (hence not being able to write in restricted directories).
By wrapping the redirection inside the `bash` execution, it ensures the whole operation is performed as `root`

Reply:
Oh, of course, that makes perfect sense :) Thanks!